RDF.js

```diff
@@ -1236,9 +1357,10 @@ function importNext(nodes, index, collections, resolve, reject) {
 			}
 			
 			var newItem = new Zotero.Item();
-			newItem.itemID = Zotero.RDF.getResourceURI(node);
+			//newItem.itemID = Zotero.RDF.getResourceURI(node);
```

Comment:
don't we need this for Zotero RDF& collection handling?

Reply:
You are possibly right! - The `itemID` lead to some problems but now I can't remember. I have to look into this again...

Reply:
Ah, now I see again. The generated ID will be different for different runs of the test cases, i.e. the test cases will constantly fail, also only the itemID is different. However, the itemID is probably important for the collections. Thus, I will revert these two changes.